script/cibuild

```diff
@@ -97,13 +97,20 @@ then
     echo "   Homebrew Python CFlags: ${HOMEBREW_PYTHON3_CLFLAGS}"
     echo "  Homebrew Python LDFlags: ${HOMEBREW_PYTHON3_LDFLAGS}"
 
+    export HOMEBREW_LIBARCHIVE_PREFIX="$(brew --prefix libarchive)"
+    export PKG_CONFIG_PATH="${HOMEBREW_LIBARCHIVE_PREFIX}/lib/pkgconfig:${PKG_CONFIG_PATH}"
+    export HOMEBREW_LIBARCHIVE_CFLAGS="-I${HOMEBREW_LIBARCHIVE_PREFIX}/include"
+    export HOMEBREW_LIBARCHIVE_LDFLAGS="-L${HOMEBREW_LIBARCHIVE_PREFIX}/lib"
```

Comment:
This is a far-less-than-optimal way to do this, and I doubt it's necessary. Just set these environment variables in the proper macOS base preset in CMakePresets.json. Then make sure that we use those presets in the CI scripts and stuff. (I think we already do on macOS, as well as on Windows.)

Reply:
Maybe the way you tried it isn't so bad after all. I'm proposing to make the necessary adjustments in CMakePresets.json, but in there, I don't think I have access to the `brew --prefix` command. I'll have to hard-code the result of that command, which is not ideal either.